clio-server/src/main/scala/org/broadinstitute/clio/server/service/SearchService.scala

```diff
@@ -7,6 +7,7 @@ import org.broadinstitute.clio.server.dataaccess.elasticsearch.{
   ElasticsearchIndex,
   ElasticsearchQueryMapper
 }
+
```

Comment:
What rules are we following for organizing our imports? It would be nice if we could use the default intellij settings for imports. In this case, I'm not sure why an empty line would be added here.

Reply:
The habitual setup I got from my last job is:
```
org.broadinstitute.*
--blank--
*other*
--blank--
scala.*
--blank--
java.*
```
But I'll revert to defaults to stop the "optimize imports" wars.